src/App.tsx

```diff
@@ -11,14 +11,17 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => ({
+      count: state.count + 1,
+    }));
```

Comment:
Just a recommendation: it would be great to use `Destructuring assignment` as much as possible 
```suggestion
    this.setState(({ count }) => ({
      count: count + 1,
    }));
```